youths/schema.py

```diff
@@ -0,0 +1,90 @@
+import graphene
+from graphene_django.types import DjangoObjectType
+from graphql_jwt.decorators import login_required
+
+from profiles.models import Profile
+
+from .consts import GENDERS, LANGUAGES
+from .models import YouthProfile
+
+
+class YouthProfileType(DjangoObjectType):
+    class Meta:
+        model = YouthProfile
+        exclude = ["id"]
+
+
+PreferredLanguage = graphene.Enum(
+    "PreferredLanguage", [(l[0].upper(), l[0]) for l in LANGUAGES]
+)
+Gender = graphene.Enum("Gender", [(g[0].upper(), g[0]) for g in GENDERS])
+
+
+class YouthProfileInput(graphene.InputObjectType):
+    ssn = graphene.String(required=True)
```

Comment:
well, technically, all of the required fields are only required when you are creating the profile. updating can be later done on separate fields as well, so I would not restrict the fields directly on the `Input ObjectType`, but would do some kind of check in the `mutate` method.